README.md

```diff
@@ -1,149 +1,109 @@
-  name: example-pipeline
+  name: config
 spec:
-  params:
-    # Customize the Params as needed by your chosen tasks
-    - name: APP_NAME
-      type: string
-      default: example
-    - name: IMAGE_PREFIX
-      type: string
-      default: "test"
-
-  workspaces:
-    - name: source
-
-  tasks:
-    # Add or Reference other tasks as needed
-    - name: example-task
-      taskRef:
-        resolver: git
-        params:
-          - name: url
-            value: https://github.com/openshift-pipelines/tektoncd-catalog.git
-          - name: revision
-            value: p
-          - name: pathInRepo
-            value: experimental/tasks/go-crane-image/v0.1.0/go-crane-image.yaml
-      workspaces:
-        - name: source
-          workspace: source
-      params:
-        - name: app
-          value: $(params.APP_NAME)
-        - name: image
-          value:
-            prefix: $(params.IMAGE_PREFIX)
+  pipeline:
+    git-resolver-config:
+      default-url: https://github.com/openshift-pipelines/tektoncd-catalog
+      default-revision: p
+      fetch-timeout: 1m 
 ```
 
-For this example we have used a PersistentVolumeClaim as follows:
+Then, on the [repositories being watched by PaC][pipelineAsCodeRepository] you can consume this catalog resources like the following example:
 
-```yaml
+```yml
 ---
-apiVersion: v1
-kind: PersistentVolumeClaim
+apiVersion: tekton.dev/v1beta1
+kind: TaskRun
 metadata:
-  labels:
-    name: test
-  name: test
+  name: task-git
 spec:
-  accessModes:
-    - ReadWriteOnce
-  resources:
-    requests:
-      storage: 250Mi
+  taskRef:
+    resolver: git
+    params:
+      - name: pathInRepo
+        value: tasks/task-git/0.1.0/task-git.yaml
 ```
 
-Then use the following commands to apply & run the above Pipeline
+Skipping the repository `url` and `revision` from `.spec.taskRef.params[]`.
 
-- Create PVC resource: ```kubectl apply -f pvc.yaml```
-- Create Pipeline: ```kubectl apply -f pipeline.yaml```
-- PipelineRun: ```tkn pipeline start example-pipeline --workspace="name=source,claimName=test,subPath=source" --showlog```
+Alternatively the same notation supported described on [Tekton Git Resolver](#tekton-git-resolver) section is supported by PaC.
 
-To learn more about resolver, use this [link](https://tekton.dev/docs/pipelines/resolution-getting-started/). 
-
-## Using Pipelines as Code
-
-Make sure all the prerequisites are present for using pac (check that [here](https://pipelinesascode.com/docs/install/getting-started/)
-
-To create a template PipelineRun resource, you can use the command: tkn pac generate
-After this it'll create a PipelineRun template, which can then be customized similar to the previous example to incorporate our tasks
-
-Some annotations to look for are: 
-```
-metadata:
-  annotations:
-    pipelinesascode.tekton.dev/on-event: "[push]"
-    pipelinesascode.tekton.dev/on-target-branch: "[main]"
-    pipelinesascode.tekton.dev/max-keep-runs: "5"
-```
+# Contributing
```

Comment:
We should link https://github.com/openshift-pipelines/tektoncd-catalog/pull/66/files#diff-f9e4af9b0893ffa7261612dc76313ee6c8eb695632ff1003623f3c083fb30f31 this here

Reply:
It is here : https://github.com/openshift-pipelines/tektoncd-catalog/pull/66/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R94. Maybe it's not super obvious ?

Reply:
We should move it up a little. Just below contributions section